extensions/amp-consent/amp-consent.md

```diff
@@ -616,7 +630,7 @@ Yes. See example [here](https://amp.dev/documentation/examples/user-consent/geol
 ##### Does AMP support the IAB TCF?
 
 AMP supports popular transparency consent frameworks including the IAB TCF v1, TCF v2 and the IAB US Privacy String.
-Please check with your consent management platform (CMP) and ad networks on their AMP support. AMP will read and pass the strings passed by the frameworks (IAB TCF v1, TCF v2 and the IAB US Privacy String) when received by CMPs/ad networks.
+Please check with your consent management platform (CMP) and ad networks on their AMP support. AMP will read and pass the strings passed by the frameworks (IAB TCF v1, TCF v2 and the IAB US Privacy String) when received by CMPs/ad networks. Read more [here](#consentmetadata).
```

Comment:
Can you add the read more to `(IAB TCF v1 ... and the IAB US Privacy String. Read more here)`

TBH I think it's fine to leave it as it is. This section was added to claim that AMP supports the TCF. Too much technical details only makes it difficult to read.